packages/server/validators/events.js

```diff
@@ -0,0 +1,229 @@
+const { param, body, check } = require("express-validator");
+
+const EventController = require("../controllers/events");
+const GuildController = require("../controllers/guilds");
+
+// validators are defined as an array of checks to perform with express-validator
+const createEventValidator = [
+  // example check for making sure guildId route parameter is a UUID
+  // modify this param("guildId") check for the other checks I listed in the PR review for "guildId"
+  param("guildId", "Invalid guild ID")
+    .trim()
+    .escape()
+    .exists({ checkFalsy: true })
+    .withMessage("guildId cannot be null or empty")
+    .isUUID()
+    .withMessage("Not a UUID")
+    .custom(async (value) => {
+      try {
+        await GuildController.getGuild(value); //Check if guild exists by getting it
+      } catch (error) {
+        //If event does not exists, then a NotFoundError is thrown from controller
+        throw new Error(error);
+      }
+    }),
+  //   add more checks for the create event route below this line as part of this array
+
+  //Title checks
+  body("title", "Invalid title")
+    .trim()
+    .escape()
+    .exists({ checkFalsy: true })
+    .withMessage("Title cannot be null or empty")
+    .isLength({ min: 1, max: 255 })
+    .withMessage("Title length must be between 1 to 255 characters"),
+
+  //Description checks
+  body("description", "Invalid description")
+    .trim()
+    .escape()
+    .isLength({ max: 255 })
+    .withMessage("Description max length is 255 characters")
+    .exists({ checkFalsy: true })
+    .withMessage("Description cannot be null or empty"),
```

Comment:
Double-check the event property checks on your validators, the only non-null event properties are eventId, guildId, and title. You don't need non-null checks on the other ones.